### PR TITLE
DCT-287

### PR DESCRIPTION
**Pull request by Craig-Elre**
Add the Wisdom For Sale tutorial and all files required for it.
Also, update the tutorial page to add WFS to it.

I did include some images that I decided not to use (reach-run-1 through reach-run-3), but could be used if it is decided that a further description of this process would be useful.

### CODE REVIEW

File: docs/src/tut/wfs/index.md [hunk 1]
```diff
@@ -0,0 +1,879 @@
...
+Interact objects enable Reach frontend and backend participants to communicate.
+:::
+
+::::
+
+::::testQ
+Name the Reach default frontend and backend files.
+
+::::testA
+`index.mjs` and `index.rsh` are the default frontend and backend filenames respectively. The Reach compiler compiles index.rsh into index.main.mjs.
```
Comment (TheChronicMonster) [inline docs/src/tut/wfs/index.md:822]: Backtick file names.

File: docs/src/tut/wfs/index.md [hunk 2]
```diff
@@ -0,0 +1,879 @@
...
+    Contract info: {"type":"BigNumber","hex":"0xa6"}
+    Your balance is 999.996 ALGO.
+    ```
+
+    The seller creates the contract, retrieves the contract information, and makes it available to the buyer who will (shortly) use the information to attach to the contract.
+	Deploying the contract cost the seller a fee.
+
+The interact objects introduced in this section facilitate communication between the frontend (e.g. `index.mjs`) and backend (e.g. `index.main.mjs`) of Reach applications, (remembering that `index.rsh` is the pre-compiled version of `index.main.mjs`).
+
+# {#wfs-16} Attach to the contract (buyer)
```
Comment (TheChronicMonster) [inline docs/src/tut/wfs/index.md:435]: This is a good section to add an intuitive explanation about interactive command-lines and how we accomplish that with the `ask` object.

File: docs/src/tut/wfs/index.md [hunk 3]
```diff
@@ -0,0 +1,879 @@
...
+
+The following learning objectives describe what you will be able to do as a result of completing this tutorial.
+
+1. Define essential Reach terms.
+1. Build a simple Reach command-line DApp.
+1. Convert the command-line DApp into a vanilla Web-app.
+
+As you build the DApp, here are some important concepts to keep in mind.
+
+## {#wfs-2} Reach Mode Diagram
```
Comment (TheChronicMonster) [inline docs/src/tut/wfs/index.md:19]: As part of the second draft, be observant of places that intuitive explanations can be inserted. Identify the topics you're teaching directly and indirectly and use those introductory moments as a way to enhance the value of the tutorial for the student. 

IMO, this guide teaches the modes of Reach
The standard library
devnet vs testnet vs mainnet
standard and atomic units
connection modes and networks
How Reach uses Docker
Attaching to a contract
payments and transfers

File: docs/src/tut/wfs/index.md [hunk 4]
```diff
@@ -0,0 +1,879 @@
...
+    ```
+
+1. In `index.rsh`, add the following before `{!rsh} exit()`:
+
+    ``` reach
+    load: /examples/docs-wisdom-12/index.rsh
+    range: 39-48
+    ```
+
+    * Line 1: The buyer always pays the contract.
```
Comment (TheChronicMonster) [inline docs/src/tut/wfs/index.md:664]: Line numbers don't match the code

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

File: docs/src/tut/wfs/index.md [hunk 5]
```diff
@@ -0,0 +1,879 @@
...
+
+    Output should resemble the following:
+
+    ```
+    The consensus network is ALGO.
+    The standard unit is ALGO
+    The atomic unit is μALGO
+    ```
+
+1. Replace your two `console.log` additions with the following, and run again:
```
[the remark this answers is not in the document]
Comment (Craig-Elre) [reply docs/src/tut/wfs/index.md:289]: That occurs in the `index.mjs` file of `docs-wisdom-4`, in `docs-wisdom-3`, I use 
`console.log(``The standard unit is ${stdlib.standardUnit}``);
console.log(``The atomic unit is ${stdlib.atomicUnit}``);`

I had to add the extra backticks above to get the code to show up as code, each one is a single backtick

**Comment by TheChronicMonster** [comment]
@Craig-Elre Several conflicting files need to be resolved; would not `make`

**Pull request closed without merging**